swagger.yaml

```diff
@@ -17,23 +17,39 @@ info:
     name: Open Service Broker API
     url: 'https://www.openservicebrokerapi.org/'
     email: open-service-broker-api@googlegroups.com
-  version: v2.13
+  version: master - might contain changes that are not yet released
```

Comment:
The tabbing looks a little off here

Reply:
I just copied it from `openapi.yaml`, do you have any suggestion?

Reply:
It looks fine in this view now - I think it's OK!